### PR TITLE
Language reworked

### PR DESCRIPTION
**Pull request by comradekingu**
## The problem

Inconsistent, unfriendly and erroneous language

## Solution

Present

## PR Status

Mergeable, ~~needs checking~~, checked, needs lgtm stamp.
@advocatux :100:  found [some things that inspired me to find more, and found yet some hard to find ones](https://translate.yunohost.org/translate/yunohost/core/nb_NO/?type=comments), hoping for a review.  

## How to test

...

## Validation

- [ ] Principle agreement 0/2 : 
- [x] Quick review 0/1 : 
- [ ] Simple test 0/1 : 
- [ ] Deep review 0/1 :

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -291,295 +291,295 @@
     "log_user_group_add": "Add '{}' group",
     "log_user_group_delete": "Delete '{}' group",
     "log_user_group_update": "Update '{}' group",
-    "log_user_update": "Update information of '{}' user",
+    "log_user_update": "Update info of '{}' user",
```
Comment (decentral1se) [inline locales/en.json]: I would keep "information" ...

File: locales/en.json [hunk 2]
```diff
@@ -291,295 +291,295 @@
...
-    "maindomain_changed": "The main domain has been changed",
-    "migrate_tsig_end": "Migration to hmac-sha512 finished",
-    "migrate_tsig_failed": "Migrating the dyndns domain {domain} to hmac-sha512 failed, rolling back. Error: {error_code} - {error}",
-    "migrate_tsig_start": "Not secure enough key algorithm detected for TSIG signature of domain '{domain}', initiating migration to the more secure one hmac-sha512",
-    "migrate_tsig_wait": "Let's wait 3min for the dyndns server to take the new key into account…",
+    "mail_alias_remove_failed": "Could not remove e-mail alias '{mail:s}'",
+    "mail_domain_unknown": "Unknown e-mail address for domain '{domain:s}'",
+    "mail_forward_remove_failed": "Could not remove e-mail forwarding '{mail:s}'",
+    "mailbox_disabled": "E-mail turned off for user {user:s}",
+    "mailbox_used_space_dovecot_down": "the Dovecot mailbox service needs to be up, if you want to fetch used mailbox space",
```
Comment (decentral1se) [inline locales/en.json]:
```suggestion
    "mailbox_used_space_dovecot_down": "The Dovecot mailbox service needs to be up, if you want to fetch used mailbox space",
```

File: locales/en.json [hunk 3]
```diff
@@ -291,295 +291,295 @@
...
+    "password_too_simple_4": "The password needs to be at least 12 characters long and contain a digit, upper, lower and special characters",
+    "path_removal_failed": "Could not remove path {:s}",
+    "pattern_backup_archive_name": "Must be a valid filename with max 30 characters, alphanumeric and -_. characters only",
     "pattern_domain": "Must be a valid domain name (e.g. my-domain.org)",
     "pattern_email": "Must be a valid email address (e.g. someone@domain.org)",
     "pattern_firstname": "Must be a valid first name",
     "pattern_lastname": "Must be a valid last name",
     "pattern_listname": "Must be alphanumeric and underscore characters only",
-    "pattern_mailbox_quota": "Must be a size with b/k/M/G/T suffix or 0 to disable the quota",
+    "pattern_mailbox_quota": "Must be a size with b/k/M/G/T suffix or 0 to not havea quota",
```
Comment (decentral1se) [inline locales/en.json]:
```suggestion
    "pattern_mailbox_quota": "Must be a size with b/k/M/G/T suffix or 0 to not have a quota",
```

File: locales/en.json [hunk 4]
```diff
@@ -72,109 +72,109 @@
...
-    "backup_applying_method_tar": "Creating the backup tar archive…",
-    "backup_archive_app_not_found": "App '{app:s}' not found in the backup archive",
-    "backup_archive_broken_link": "Unable to access backup archive (broken link to {path:s})",
-    "backup_archive_mount_failed": "Mounting the backup archive failed",
-    "backup_archive_name_exists": "The backup's archive name already exists",
+    "backup_applying_method_tar": "Creating the backup TAR archive…",
+    "backup_archive_app_not_found": "Could not find the app '{app:s}' in the backup archive",
+    "backup_archive_broken_link": "Could not access the backup archive (broken link to {path:s})",
+    "backup_archive_mount_failed": "Could not mounting the backup archive",
+    "backup_archive_name_exists": "A backup archive with this name already exists.",
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply locales/en.json]: I know, but that exact string exists elsewhere with minor changes, so couldn't help myself. Doing them all is for another PR.

File: locales/en.json [hunk 5]
```diff
@@ -72,109 +72,109 @@
...
-    "backup_archive_open_failed": "Unable to open the backup archive",
-    "backup_archive_system_part_not_available": "System part '{part:s}' not available in this backup",
-    "backup_archive_writing_error": "Unable to add files '{source:s}' (named in the archive: '{dest:s}') to backup into the compressed archive '{archive:s}'",
-    "backup_ask_for_copying_if_needed": "Some files couldn't be prepared to be backuped using the method that avoid to temporarily waste space on the system. To perform the backup, {size:s}MB should be used temporarily. Do you agree?",
-    "backup_borg_not_implemented": "Borg backup method is not yet implemented",
-    "backup_cant_mount_uncompress_archive": "Unable to mount in readonly mode the uncompress archive directory",
-    "backup_cleaning_failed": "Unable to clean-up the temporary backup directory",
+    "backup_archive_open_failed": "Could not open the backup archive",
+    "backup_archive_system_part_not_available": "System part '{part:s}' unavailable in this backup",
+    "backup_archive_writing_error": "Could not add the add files '{source:s}' (named in the archive: '{dest:s}') to be backed up into the compressed archive '{archive:s}'",
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply locales/en.json]: You have a doublespace and an unmarked noun, will fix.

File: locales/en.json [hunk 6]
```diff
@@ -72,109 +72,109 @@
...
+    "backup_ask_for_copying_if_needed": "Some files could not be prepared for bacup using the method that avoids temporarily wasting space on the system. To perform the backup, {size:s}MB will be temporarily. Do you agree?",
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply locales/en.json]: My "k"-key is a bit broken, PEBKAK ;)

File: locales/en.json [hunk 7]
```diff
@@ -72,109 +72,109 @@
...
-    "certmanager_no_cert_file": "Unable to read certificate file for domain {domain:s} (file: {file:s})",
-    "certmanager_self_ca_conf_file_not_found": "Configuration file not found for self-signing authority (file: {file:s})",
-    "certmanager_unable_to_parse_self_CA_name": "Unable to parse name of self-signing authority (file: {file:s})",
-    "confirm_app_install_warning": "Warning: this application may work but is not well-integrated in YunoHost. Some features such as single sign-on and backup/restore might not be available. Install anyway? [{answers:s}] ",
+    "certmanager_cert_install_success": "Let's Encrypt certificate now installed for the domain '{domain:s}'",
+    "certmanager_cert_install_success_selfsigned": "Self-signed certificate now installed for the domain '{domain:s}'",
+    "certmanager_cert_renew_success": "Let's Encrypt certificate renewed for the domain '{domain:s}'",
+    "certmanager_cert_signing_failed": "Could not sign the new certificate",
+    "certmanager_certificate_fetching_or_enabling_failed": "It seems actually using the new certificate for {domain:s} did not work…",
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply locales/en.json]: Everything works up until that point, so tried to make that the issue. I think the cert might be OK but possibly still fails at this point (?), which I was going for.

File: locales/en.json (hunk 1 above)
Comment (comradekingu) [reply locales/en.json]: "info" should be in consistent use, it saves on space similar to "admin" over administrator.

File: locales/en.json (hunk 2 above)
Comment (comradekingu) [reply locales/en.json]: Nice catch :)

File: locales/en.json (hunk 3 above)
Comment (comradekingu) [reply locales/en.json]: havea :+1:  :)

File: locales/en.json [hunk 8]
```diff
@@ -72,109 +72,109 @@
...
-    "backup_borg_not_implemented": "Borg backup method is not yet implemented",
-    "backup_cant_mount_uncompress_archive": "Unable to mount in readonly mode the uncompress archive directory",
-    "backup_cleaning_failed": "Unable to clean-up the temporary backup directory",
+    "backup_archive_open_failed": "Could not open the backup archive",
+    "backup_archive_system_part_not_available": "System part '{part:s}' unavailable in this backup",
+    "backup_archive_writing_error": "Could not add the add files '{source:s}' (named in the archive: '{dest:s}') to be backed up into the compressed archive '{archive:s}'",
+    "backup_ask_for_copying_if_needed": "Some files could not be prepared for bacup using the method that avoids temporarily wasting space on the system. To perform the backup, {size:s}MB will be temporarily. Do you agree?",
+    "backup_borg_not_implemented": "The Borg backup method is not yet implemented",
+    "backup_cant_mount_uncompress_archive": "Could not mount the uncompressed archive write protected",
```
[the remark this answers is not in the document]
Comment (Psycojoker) [reply locales/en.json]: I think "which is" is better here but apart from that : yes we need this kind of change in this contexte.

File: locales/en.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (Psycojoker) [reply locales/en.json]: Unrelated to which is the better way of saying that but I'm really wondering if the first sentence might not be a bit to "advanced english" for our general users? I might be mistaking but I don't find this formulation to be the more obvious (talking as a french native speaker)

File: locales/en.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (decentral1se) [reply locales/en.json]: :+1:

File: locales/en.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (comradekingu) [reply locales/en.json]: But it isn't, the original is "failing to mount it in readonly mode". I will add something to make the action of making it so clear.

File: locales/en.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (comradekingu) [reply locales/en.json]: That is very true, my bad. Generally I opt for the best language for each respective language, including translations, but my initial attempt is needless complexity, and will result in bad translations.

File: locales/en.json [hunk 9]
```diff
@@ -2,67 +2,67 @@
...
+    "app_upgrade_app_name": "Now upgrading {app}…",
+    "app_upgrade_failed": "Could not upgrade {app:s}",
+    "app_upgrade_some_app_failed": "Some applications could not be upgraded",
+    "app_upgraded": "{app:s} upgraded",
     "apps_permission_not_found": "No permission found for the installed apps",
-    "apps_permission_restoration_failed": "Permission '{permission:s}' for app {app:s} restoration has failed",
-    "appslist_corrupted_json": "Could not load the application lists. It looks like {filename:s} is corrupted.",
-    "appslist_could_not_migrate": "Could not migrate app list {appslist:s}! Unable to parse the url… The old cron job has been kept in {bkp_file:s}.",
-    "appslist_fetched": "The application list {appslist:s} has been fetched",
+    "apps_permission_restoration_failed": "Grant the permission permission '{permission:s}' to restore {app:s}",
```
Comment (alexAubin) [inline locales/en.json]:
```suggestion
    "apps_permission_restoration_failed": "Could not restore the permission '{permission:s}' while restoring application {app:s}",
```

(though many messages related to permission will change in https://github.com/YunoHost/yunohost/pull/795 :/ )

File: locales/en.json [hunk 10]
```diff
@@ -72,109 +72,109 @@
...
-    "backup_borg_not_implemented": "Borg backup method is not yet implemented",
-    "backup_cant_mount_uncompress_archive": "Unable to mount in readonly mode the uncompress archive directory",
-    "backup_cleaning_failed": "Unable to clean-up the temporary backup directory",
+    "backup_archive_open_failed": "Could not open the backup archive",
+    "backup_archive_system_part_not_available": "System part '{part:s}' unavailable in this backup",
+    "backup_archive_writing_error": "Could not add the files '{source:s}' (named in the archive '{dest:s}') to be backed up into the compressed archive '{archive:s}'",
+    "backup_ask_for_copying_if_needed": "Some files could not be prepared for backup using the method that avoids temporarily wasting space on the system. To perform the backup, {size:s}MB will be temporarily. Do you agree?",
+    "backup_borg_not_implemented": "The Borg backup method is not yet implemented",
+    "backup_cant_mount_uncompress_archive": "Could not mount the uncompressed archive as write protected",
```
Comment (alexAubin) [inline locales/en.json]:
```suggestion
    "backup_cant_mount_uncompress_archive": "Could not mount the uncompressed archive as read-only",
```

File: locales/en.json [hunk 11]
```diff
@@ -72,109 +72,109 @@
...
-    "backup_custom_mount_error": "Custom backup method failure on 'mount' step",
-    "backup_custom_need_mount_error": "Custom backup method failure on 'need_mount' step",
-    "backup_delete_error": "Unable to delete '{path:s}'",
-    "backup_deleted": "The backup has been deleted",
-    "backup_extracting_archive": "Extracting the backup archive…",
-    "backup_hook_unknown": "Backup hook '{hook:s}' unknown",
-    "backup_invalid_archive": "Invalid backup archive",
-    "backup_method_borg_finished": "Backup into borg finished",
-    "backup_method_copy_finished": "Backup copy finished",
+    "backup_creation_failed": "Could not create the backup creation",
```
Comment (alexAubin) [inline locales/en.json]:
```suggestion
    "backup_creation_failed": "Could not create the backup archive",
```

File: locales/en.json [hunk 12]
```diff
@@ -291,295 +291,295 @@
...
+    "migrations_pending_cant_rerun": "Those migrations are still pending, so cannot be run again: {ids}",
     "migrations_running_forward": "Running migration {id}…",
     "migrations_skip_migration": "Skipping migration {id}…",
-    "migrations_success_forward": "Successfully ran migration {id}!",
-    "migrations_to_be_ran_manually":  "Migration {id} has to be ran manually. Please go to Tools > Migrations on the webadmin, or run `yunohost tools migrations migrate`.",
-    "monitor_disabled": "The server monitoring has been disabled",
-    "monitor_enabled": "The server monitoring has been enabled",
-    "monitor_glances_con_failed": "Unable to connect to Glances server",
-    "monitor_not_enabled": "Server monitoring is not enabled",
+    "migrations_success_forward": "Ran migration {id}",
```
Comment (alexAubin) [inline locales/en.json]:
```suggestion
    "migrations_success_forward": "Migration {id} completed",
```

File: locales/en.json [hunk 13]
```diff
@@ -291,295 +291,295 @@
...
+    "service_description_avahi-daemon": "allows you to reach your server using 'yunohost.local' in your local network",
+    "service_description_dnsmasq": "Handles domain name resolution (DNS)",
+    "service_description_dovecot": "Allows e-mail clients to access/fetch email (via IMAP and POP3)",
+    "service_description_fail2ban": "Protects against brute-force and other kinds of attacks from the Internet",
+    "service_description_glances": "Monitors system info on your server",
+    "service_description_metronome": "Manage XMPP instant messaging accounts",
+    "service_description_mysql": "Stores applications data (SQL database)",
+    "service_description_nginx": "Serves or provides access to all the websites hosted on your server",
+    "service_description_nslcd": "Handles YunoHost user shell connection",
+    "service_description_php7.0-fpm": "runs applications written in PHP with NGINX",
```
Comment (alexAubin) [inline locales/en.json]:
```suggestion
    "service_description_php7.0-fpm": "Runs applications written in PHP with NGINX",
```

File: locales/en.json [hunk 14]
```diff
@@ -291,295 +291,295 @@
...
-    "service_disable_failed": "Unable to disable service '{service:s}'\n\nRecent service logs:{logs:s}",
-    "service_disabled": "The service '{service:s}' has been disabled",
-    "service_enable_failed": "Unable to enable service '{service:s}'\n\nRecent service logs:{logs:s}",
-    "service_enabled": "The service '{service:s}' has been enabled",
+    "service_add_failed": "Could not add the service '{service:s}'",
+    "service_added": "The service '{service:s}' added",
+    "service_already_started": "The service '{service:s}' has already been started",
+    "service_already_stopped": "The service '{service:s}' has already been stopped",
+    "service_cmd_exec_failed": "Could not execute the command '{command:s}'",
+    "service_description_avahi-daemon": "allows you to reach your server using 'yunohost.local' in your local network",
```
Comment (alexAubin) [inline locales/en.json]:
```suggestion
    "service_description_avahi-daemon": "Allows you to reach your server using 'yunohost.local' in your local network",
```

File: locales/en.json [hunk 15]
```diff
@@ -291,295 +291,295 @@
...
-    "user_creation_failed": "Unable to create user",
-    "user_deleted": "The user has been deleted",
-    "user_deletion_failed": "Unable to delete user",
-    "user_home_creation_failed": "Unable to create user home folder",
-    "user_info_failed": "Unable to retrieve user information",
-    "user_not_in_group": "User {user:s} not in group {group:s}",
+    "upnp_disabled": "UPnP turned off",
+    "upnp_enabled": "UPnP turned on",
+    "upnp_port_open_failed": "Could not open port via UPnP",
+    "user_already_in_group": "The User '{user:}' already in the group {group:s}",
```
Comment (alexAubin) [inline locales/en.json]:
```suggestion
    "user_already_in_group": "The user '{user:}' is already in the group {group:s}",
```

File: locales/en.json [hunk 16]
```diff
@@ -2,67 +2,67 @@
...
     "app_not_correctly_installed": "{app:s} seems to be incorrectly installed",
-    "app_not_installed": "The application '{app:s}' is not installed. Here is the list of all installed apps: {all_apps}",
+    "app_not_installed": "The application '{app:s}' is not to be found among all installed apps: {all_apps}",
     "app_not_properly_removed": "{app:s} has not been properly removed",
     "app_package_need_update": "The app {app} package needs to be updated to follow YunoHost changes",
-    "app_removed": "{app:s} has been removed",
+    "app_removed": "{app:s} removed",
     "app_requirements_checking": "Checking required packages for {app}…",
-    "app_requirements_failed": "Unable to meet requirements for {app}: {error}",
+    "app_requirements_failed": "Did not meet requirements for {app}: {error}",
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply locales/en.json]: @alexAubin Does this mean dependencies?

File: locales/en.json (hunk 12 above)
Comment (comradekingu) [reply locales/en.json]: Nice.

File: locales/en.json (hunk 9 above)
Comment (comradekingu) [reply locales/en.json]: How I read this is a permission is needed to restore the app, meaning one could grant said permission, and the app would be restored. The new active suggestion instructs the user in solving the problem. Your change seems to question the permission as the only thing that could not be restored, but does not detail why, or what to do about it.

Comment (alexAubin) [reply locales/en.json]: Uuuuh not sure to understand your point but anyway this string is to be removed in https://github.com/YunoHost/yunohost/pull/795 :| ...

**Comment by alexAubin** [comment]
As dicussed on the chat yesterday, yolomerging so that we can keep on working on other PR that affects a lot of strings, such as https://github.com/YunoHost/yunohost/pull/793 and https://github.com/YunoHost/yunohost/pull/795

**Pull request merged**